media/js/firefox/accounts-2019.js

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
```

Comment:
Cleanup: empty file can go.

Reply:
Yeah, I ended up not even needing any special JS. Forgot to clean this up (and the bundle).